src/environment_setup/conda.mli

```diff
@@ -1,13 +1,20 @@
 open Biokepi_run_environment
 
+type python_version_type = [
+  | `Python2
+  | `Python3
+  | `PythonCustom of string
+  | `PythonToolDependency of string
+]
+
```

Comment:
Why not use a regular variant? I would also document this type, as the meaning these variants isn't obvious.